src/ClpCInterface.jl

```diff
@@ -620,13 +620,13 @@ function number_iterations(model::ClpModel)
     _jl__check_model(model)
     @clp_ccall numberIterations Int32 (Ptr{Cvoid},) model.p
 end
-get_iteration_count = number_iterations
+const get_iteration_count = number_iterations
 
 # Set number of iterations
 function set_number_iterations(model::ClpModel, iters::Integer)
     _jl__check_model(model)
     @clp_ccall setNumberIterations Cvoid (Ptr{Cvoid}, Int32) model.p iters
-    return
+    return nothing
```

Comment:
Use `return` instead of `return nothing`: http://www.juliaopt.org/JuMP.jl/dev/style/#Return-statements-1.

Reply:
ok sorry I had missed this part of the style guide